src/main/java/com/kdt/localinfo/post/controller/PostController.java

```diff
@@ -24,4 +24,22 @@ public PostController(PostService postService) {
         Long savedPostId = postService.createPost(postDto);
         return ApiResponse.successCreated(savedPostId);
     }
+
+    @GetMapping("/posts/{postId}")
+    public ApiResponse<PostDto> findDetailPost(@PathVariable Long postId) throws NotFoundException {
+        PostDto foundPost = postService.findDetailPost(postId);
+        return ApiResponse.ok(foundPost);
+    }
+
+    @GetMapping("/categories/{categoryId}")
```

Comment:
posts를 반환하는 거면 아래와 같이 하는게 좋지 않을까요?
```suggestion
    @GetMapping("posts/categories/{categoryId}")
```